l10n_ch_invoice_reports/models/report.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from io import BytesIO
+import PyPDF2
+from odoo import models
+
+
+class IrActionsReport(models.Model):
+    _inherit = "ir.actions.report"
+
+    def merge_invoice_pdf_in_memory(self, docs):
```

Comment:
why you cannot use a default method merge_pdf_in_memory?

Reply:
If by 'default method' you mean method in ir.actions.report class, then there is no such method in this class. Otherwise please specify what do you mean by 'default'.

Reply:
ah it's defined in this repo as well
https://github.com/OCA/l10n-switzerland/blob/83ec9324741cfd76def3ae654211854fb332330c/l10n_ch_payment_slip/report/ir_action.py#L151
then I will prefer have name "merge_pdf_in_memory"